dev/FlipView/TestUI/FlipViewPage.xaml

```diff
@@ -9,11 +9,25 @@
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
     mc:Ignorable="d">
 
-    <Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}" Margin="12">
-        <FlipView Width="200" Height="200">
+    <StackPanel Background="{ThemeResource ApplicationPageBackgroundThemeBrush}" Margin="12">
+        <FlipView Width="200" Height="200"
+                  CornerRadius="2">
```

Comment:
Use conditional Xaml here.

We should make it so that our test app is always usable on all supported platforms. It is ok if certain features on a Page are missing/don't work. But just navigating around in the app should not cause a crash.